docs/src/modules/ROOT/pages/integration/config-solver-properties.adoc

```diff
@@ -0,0 +1,61 @@
+////
+Quarkus and Spring Boot support the same configuration properties.
+All the properties are in this file, which can then be included multiple times.
+The {property_prefix} attribute is used for Quarkus properties.
+////
+
+{property_prefix}timefold.solver.<solverName>.solver-config-xml::
```

Comment:
I'm thinking... this is content duplication with the `config-properties.adoc` file.
Can we somehow define a property (just like we do `property_prefix`) which will only inject `<solverName>.` when necessary by the context?

That way, we'll be able to use the same file for all three places where it's used.

Reply:
Let me check it

Reply:
That was a great idea!
It is possible to address your request.